udata_datalake_service/background_tasks.py

```diff
@@ -15,7 +17,7 @@
 
 BROKER_URL = os.environ.get("BROKER_URL", "redis://localhost:6380/0")
 MINIO_FOLDER = os.environ.get("MINIO_FOLDER", "folder")
-celery = Celery('tasks', broker=BROKER_URL)
+celery = Celery("tasks", broker=BROKER_URL)
 
 
 def download_resource(url):
```

Comment:
Can we add in this function a test of the size of the resource.
We can set up a new env variable `MAX_FILESIZE_ALLOWED` (to 1Go for instance) and break out of the loop if we exceed the limit. 
We still need, even if the download does not succeed, a message in `resource.analyzed` a custom message that indicates that we could not download file because of size limitation.